### ISSUE TITLE
With the same data and same model, the effect of Gaussian probability prediction becomes worse

### ISSUE THREAD

**Issue by Luffffffy** [unit8co/darts#1427]
And Loss is a negative number, is it possible？

**Comment by dennisbader** [comment]
Could you try be more specific in describing the issue? 
Also, please add a minimum working example to reproduce the issue.

**Comment by Luffffffy** [comment]
> Could you try be more specific in describing the issue? Also, please add a minimum working example to reproduce the issue.

I may not be able to reproduce it quickly with a small amount of work because mine is complex, but for sure, I use the `NHiTSModel `model, trained on my dataset, and I get a deterministic prediction , Then I used the parameters of `GaussianLikelihood` and sampled 1000 times, so I got a prediction based on probability, but I found that during training, the loss became a negative number, and I got the definite value prediction in the Likelihood output and before The deterministic predictions produced by the model without the Likelihood parameter were compared, and the effect was found to be worse

**Comment by hrzn** [comment]
Two small points here
* It is expected for the loss to take negative values for NLL losses such as that used by GaussianLikelihood
* You will sometimes get better accuracy metrics using point forecasts (and corresponding losses). It all depends on the data, metrics and loss functions being used.
